_config.yml

```diff
@@ -21,6 +21,7 @@ sidebar_categories:
     - docs/advanced/theme/wysiwyg
     - docs/advanced/theme/analytics
     - docs/advanced/theme/translations
+    - docs/advanced/theme/sitemap
```

Comment:
It feels weird to have it in the theme section since it is mostly GraphQL work…

I propose to relocate it under the "Production ready" section.